resources/prosody-plugins/mod_filter_iq_jibri.lua

```diff
@@ -10,12 +15,30 @@ module:hook("pre-iq/full", function(event)
         if jibri then
             local session = event.origin;
             local token = session.auth_token;
+            local feature = jibri.attr.recording_mode == 'file' and 'recording' or 'livestreaming';
+            local is_allowed = is_feature_allowed(session.jitsi_meet_context_features, feature);
+
+            -- if current user is not allowed, but was granted moderation by a user
+            -- that is allowed by its features we want to allow it
+            local is_granting_allowed = false;
```

Comment:
I don't think you care about the distinction of whether it was "naturally" allowed or because the features were granted. I suggest simplifying to something like
```
local is_allowed = is_feature_allowed(session.jitsi_meet_context_features, feature) or
is_feature_allowed(session.granted_jitsi_meet_context_features, feature)
```

Reply:
But I want to check that only if there are features set.

Reply:
A right, we have changed behavior of is_feature_allowed, so this will work.

Reply:
Nope, this is not correct as it will check for moderator rights if you have the feature = "false". 
so we have two cases, are features available and is the feature set to false.